commit ed5ff9b9505e50b545e86dfbdd32077f0ddda0cb
Author: Taylor Jakobson <tjakobs@us.ibm.com>
Date:   Thu Feb 1 14:44:32 2018 -0600

    Use signed char instead of char
    
    Python3 does not have the "character" type, use signed char instead.

diff --git a/targetcli/ui_backstore.py b/targetcli/ui_backstore.py
index 3a2db10..5af448f 100644
--- a/targetcli/ui_backstore.py
+++ b/targetcli/ui_backstore.py
@@ -536,7 +536,7 @@ class UIBlockBackstore(UIBackstore):
         except (OSError, IOError):
             raise ExecutionError("Could not open %s" % dev)
         # ioctl returns an int. Provision a buffer for it
-        buf = array.array('c', [chr(0)] * 4)
+        buf = array.array('b', [0] * 4)
         try:
             fcntl.ioctl(f, BLKROGET, buf)
         except (OSError, IOError):
